CHANGELOG.md

```diff
@@ -4,7 +4,13 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.6.0 (unreleased)
+## 1.6.0 (2020-01-14)
+
+This release adds major new features and bugfixes since the 1.5.1 release.
+We deem it moderate priority for upgrading.
+
+This release allows keeping aggregate data of continuous aggregates when
```

Comment:
The major new feature in this release allows users to keep the aggregated data in a continuous aggregate while dropping the raw data with drop_chunks. This allows users to save storage by keeping only the aggregates. This allows user to enforce more complex data management policies and control costs.